Tutorials/start/configuring-faraday.md

```diff
@@ -11,22 +11,23 @@ The application files are located:
 > NOTE: Device Configuration using the application tool in this tutorial is currently limited to programming a single device at a time. Also, this process will be automated in the future.
 
 ## Device Configuration Setup
-
-First we should ensure the Device Configuration program can communicate with Proxy running in the background. For this we need to edit the `[DEVICES]` section of `deviceconfiguration.ini` to contain the same configuration Proxy is configured with in `proxy.ini`
-
+ 
+First, we should ensure the Device Configuration program can communicate with Proxy running in the background. For this we need to create the `deviceconfiguration.ini` file from `deviceconfiguration.sample.ini` as we did for Proxy.  Next, edit the `[DEVICES]` section to contain the same configuration Proxy is configured with in `proxy.ini`
+ 
 `[FLASK]`: Flask server configuration values
  * `HOST`: IP Address or hostname of flask server
  * `PORT`: Network port to serve data
-
-
+ 
+ 
 `[DEVICES]`: Unit 0 Proxy configuration values section
```

Comment:
Non-blocking as this is an error from outside this PR. Would be good to remove reference to Unit 0 Proxy configuration as this is all unit configuration. Reword as you see fit if you like.